android/app/src/main/java/app/organicmaps/downloader/DownloaderNotifier.java

```diff
@@ -58,7 +58,7 @@ public static void notifyDownloadFailed(@NonNull Context context, @Nullable Stri
     final int FLAG_IMMUTABLE = Build.VERSION.SDK_INT < Build.VERSION_CODES.M ? 0 : PendingIntent.FLAG_IMMUTABLE;
     final Intent contentIntent = MwmActivity.createShowMapIntent(context, countryId);
     contentIntent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-    final PendingIntent contentPendingIntent = PendingIntent.getActivity(context, 0, contentIntent,
+    final PendingIntent contentPendingIntent = PendingIntent.getActivity(context, 100, contentIntent,
```

Comment:
Thanks! What do these codes/integers mean? Are they specified somewhere else? Do they need some constants?

Another question: what are the steps to reproduce the issue you're fixing? Can you please write them in the PR description (or create an issue)?

Reply:
I updated the PR description. These numbers are pretty much arbitrary and identify pending intents. If they're set to the same number and the target intent is the same, they will be overwritten. This change makes the 3 pending intents take separate identities instead.
If you think that using constants will be more clear, that's a valid concern and I'm going to update the PR.